pandapipes/pf/result_extraction.py

```diff
@@ -49,8 +49,7 @@ def extract_all_results(net, nodes_connected, branches_connected):
         }
         branch_results.update(gas_branch_results)
     for comp in net['component_list']:
-        comp.extract_results(net, net["_options"], branch_results, nodes_connected,
-                             branches_connected)
+        comp.extract_results(net, net["_options"], branch_results, "hydraulics")
```

Comment:
Because inside this if-condition, we only look at hydraulics. It is not related to the simulation mode, we need to extract the hydraulic results here, the heat transfer results later.

Reply:
Was updated.